src/users/users.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@nestjs/common';
+import { User } from './interfaces/user.interface';
+
+@Injectable()
+export class UsersService {
+  private readonly users: User[] = [];
+
+  findOne(): User {
+    return {
+      id: '1',
+      name: 'zero!',
+      secret: 'anaSecret',
```

Comment:
I am not sure if the secret should have a minimum length or specefic encoding, but we were using this in [alpha-client](https://github.com/tazkrtak/alpha-client/blob/97a17669875fd6005eafd36226907397022b886f/lib/util/totp.dart#L26-L30). You can use `WKSFANU2PIIOF5GK` as a secret, just to be sure nothing fails when we use it with the TOTP algorithm.